src/extension-support/extension-manager.js

```diff
@@ -310,15 +310,22 @@ class ExtensionManager {
     _prepareMenuInfo (serviceName, menus) {
         const menuNames = Object.getOwnPropertyNames(menus);
         for (let i = 0; i < menuNames.length; i++) {
-            const item = menuNames[i];
+            const menuName = menuNames[i];
```

Comment:
Maybe add a comment here saying we're normalizing the menu definitions to have the structure `{items: [...]}`.